ulohy_05/povinna_uloha_6_zviratka.py

```diff
@@ -0,0 +1,30 @@
+zvirata = ["pes", "kočka", "králík","had"]
+zvirata.append("andulka")
+
+# Slovník ze seznamu, klíč je druhý znak ve slově a hodnota je slovo
+zvirata_dict = {}
+for zvire in zvirata:
+    zvirata_dict[zvire[1]] = zvire
+
+# Seřazení klíčů
+keys = zvirata_dict.keys()
+sort_letters = sorted(keys)
+
+# Vytvoření nového slovníku dle seřazených klíčů
+sort_value = {}
+for key in sort_letters:
+    sort_value[key] = zvirata_dict[key]
+
+# Vypsání pouze hodnot slovníku jako seznam
+seznam_zvirat = list(zvirata_dict.values())
```

Comment:
Není zbytečné vytvářet opět nový slovník a z něho pak list, když nakonec chceme list?

Reply:
Ve chvíli, kdy jsem to dělala, mě nenapadlo, že by šel udělat rovnou seznam. Proto jsem šla přes nový slovník.